lib/containers/container_images.pm

```diff
@@ -57,6 +57,7 @@ sub build_container_image {
 sub test_containered_app {
     my %args       = @_;
     my $runtime    = $args{runtime};
+    my $buildah    = $args{buildah} // 0;
```

Comment:
Looks confusing to me. Having a runtime (which can be `buildah`) but then a boolean which say buildah or not. Maybe a good reason to add some perl POD.

Reply:
Done